src/fu-util.c

```diff
@@ -2382,6 +2382,63 @@ fu_util_check_polkit_actions (GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_generate_motd (FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	g_autoptr(GPtrArray) devices = NULL;
+	guint upgrade_count = 0;
+	g_autofree gchar *upgrade_str = NULL;
```

Comment:
`g_auto`* all at the end of the C99 block please.

Reply:
OK, fixed up.